Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -122,6 +125,32 @@ class MemoryViewTable final : public QTableWidget
     }
   }
 
+  void OnItemChanged(QTableWidgetItem* item)
+  {
+    if (m_view->m_updating == true)
+      return;
+
+    m_view->m_updating = true;
+    QString text = item->text();
+    u32 address = item->data(USER_ROLE_CELL_ADDRESS).toUInt();
+    int type = item->data(USER_ROLE_VALUE_TYPE).toInt();
+    std::vector<u8> bytes;
+
+    bytes = m_view->ConvertTextToBytes(type, text);
+
+    // Cannot be used while update is running.
```

Comment:
What is this comment trying to say here?

Reply:
I was just noting this caused a stack overflow with Update().

Reply:
Yeah, but what is it referring to? The `OnItemChanged()` function? Then it should be on top of that, not in the middle of it.

Reply:
I think I can just remove the note now. The accessor itself was crashing with the accessor in Update() I think? When I deleted the accessor stuff, it was fine, so I was marking that specifically.